ci/scripts/release.sh

```diff
@@ -71,8 +71,8 @@ if [ "${ARCH}" == "aarch64" ]; then
   # see https://github.com/tikv/jemallocator/blob/802969384ae0c581255f3375ee2ba774c8d2a754/jemalloc-sys/build.rs#L218
   export JEMALLOC_SYS_WITH_LG_PAGE=16
 fi
-cargo build -p risingwave_cmd_all --features "rw-static-link" --profile release
-cargo build -p risingwave_cmd --bin risectl --features "rw-static-link" --profile release
+cargo build -p risingwave_cmd_all --features "rw-static-link" --features all-udf --profile release
```

Comment:
Interesting. Does it mean that we used to release versions without support for embedded 
UDF?

Reply:
I'm considering whether we should make it a default feature but always use `--no-default-features` for RiseDev builds, so that we can avoid such mistakes in the future...

Reply:
Oh yes, the released version did not contain python UDF and deno UDF.

Reply:
But making these feature as default will cause them to be built by rust-analyzer...

Reply:
> I'm considering whether we should make it a default feature but always use `--no-default-features` for RiseDev builds, so that we can avoid such mistakes in the future...

I don't think it's worth it:
- When release failed, we can just fix it. And it's ok if the script is dirty. We just need to get it done right once.
- But `default` features is to make developers happy, which can be hard to fix.

Reply:
> But making these features as default will cause them to be built by rust-analyzer...

Can also suggest developers to set `rust-analyzer.cargo.noDefaultFeatures` in the editor. 😕

Reply:
> The point is the most of developers do not pay much attention to the compile time. If something comes naturally, it's good. If it requires extra effort, they might give up.

I 100% agree. However, my opinion from this fact is that it's better to offer "free" fast compile time for developers. If they are not good at it, we'd better make the default faster. Saving every one's time does good to the whole world!

Reply:
> how to enable these features, especially the IDE part

I'm thinking whether it's possible to offer IDE experience without config for feature flags. e.g., `cfg_or_panic(madsim)`.

One idea is that: we put each connector impl in separated crate, which is an optional dependency. If so, maybe the IDE can work without config.

Reply:
Let's go to https://github.com/risingwavelabs/risingwave/issues/16841